finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java

```diff
@@ -28,14 +31,17 @@
 public class InventoryManager {
```

Comment:
Are the changes in this file only meant for the `finish` directory? The `start` directory's `InventoryManager` doesn't have these changes while all other files do.

Reply:
The Start directory shouldn't have an InventoryManager, the user creates it during the guide.

Reply:
ah ok, nvm then :P